ipywidgets/widgets/widget_int.py

```diff
@@ -150,6 +150,7 @@ class IntSlider(_BoundedInt):
         default_value='horizontal', help="Vertical or horizontal.").tag(sync=True)
     _range = Bool(False, help="Display a range selector").tag(sync=True)
     readout = Bool(True, help="Display the current value of the slider next to it.").tag(sync=True)
+    readout_format = Unicode('d', help="Format for the readout").tag(sync=True)
```

Comment:
I was thinking you could just add one line and a blank line in the existing docstring for the IntSlider class (line 145 above). Not to document all of the attributes, but just as a "FYI, this particular attribute uses this format.." type of line.